software_reseller/data/project_project.xml

```diff
@@ -0,0 +1,68 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<odoo>
+    <data noupdate="1">
+        <record id="software_reseller.project_project_1_c7587515" model="project.project" >
+            <field name="type_ids" eval="[(6, 0, [ref('software_reseller.project_task_type_1_b7e703fd'), ref('software_reseller.project_task_type_2_06f38fd6'), ref('software_reseller.project_task_type_3_4f5b5674'), ref('software_reseller.project_task_type_4_4c8b36cf')])]"/>
+            <field name="task_properties_definition" eval="[{'name': '0199a697eee37d67', 'type': 'many2one', 'domain': False, 'string': 'Client', 'comodel': 'res.partner', 'default': False}, {'name': '92523a6947203790', 'type': 'char', 'string': 'Java Version', 'default': '20'}, {'name': 'b2e92295f220408e', 'type': 'char', 'string': 'License Key', 'default': ''}]"/>
+            <field name="label_tasks">Licenses</field>
+            <field name="favorite_user_ids" eval="[(6, 0, [])]"/>
+            <field name="name">Java Licenses</field>
+            <field name="timesheet_product_id" eval="False"/>
+            <field name="stage_id" ref="project.project_project_stage_0"/>
+            <field name="user_id" ref="base.user_admin"/>
+            <field name="rating_request_deadline">2023-09-07 10:13:18</field>
```

Comment:
field rating_request_deadline is computed, and this date is wrong, same for next records